src/toil_scripts/exome_variant_pipeline/README.md

```diff
@@ -14,7 +20,7 @@ pipeline can be run on it's own. If preprocessing is selected, it will always oc
 This pipeline produces a tarball (tar.gz) file for a given sample that contains:
 
     MuTect: Mutect.vcf, Mutect.cov, Mutect.out
-    Pindel: 
+    Pindel: Pindel output files
```

Comment:
Ya, cheaped out b/c i couldn't find my test folder with all of pindels output. I could just list [these](http://seqanswers.com/forums/showpost.php?p=129742&postcount=9).